assets/src/js/admin/reports/components/rest-chart/index.js

```diff
@@ -29,20 +47,55 @@ const RESTChart = ( { type, aspectRatio, endpoint, showLegend } ) => {
 				},
 			} )
 				.then( function( response ) {
+					setQuerying( false );
+					setLoaded( true );
 					setFetched( response.data.data );
+					if ( endpoint === 'income' ) {
+						const status = response.data.data.status;
+						dispatch( {
+							type: 'SET_GIVE_STATUS',
+							payload: status,
+						} );
+						dispatch( {
+							type: 'SET_PAGE_LOADED',
+							payload: true,
+						} );
```

Comment:
Convert these to actions in the reducer and import them.

Reply:
Done, ref: https://github.com/impress-org/givewp/commit/bd2900485ab5e8a162ce4825c17e65e87a30cc15